### PR TITLE
[skip ci] simplified default issue template

### PR DESCRIPTION
**Pull request by hackaugusto**
Changed our issue template to target our users instead of being a template for our description of tasks.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
...
-Bother with this section only if you are reporting a bug
+Here add a detailed description of your system, e.g. output of the following script:
 
-### System Description
-Here add a detailed description of your system, e.g. output of `raiden version`, `geth version`, `solc --version` e.t.c.
+```
+uname -a
+command -v solc && solc --version
+command -v geth && geth version
+command -v parity && partiy --version
```
Comment (LefterisJP) [inline .github/ISSUE_TEMPLATE]:
```diff
-command -v parity && partiy --version
+command -v parity && parity --version
```

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
...
+command -v raiden && raiden version
```
Comment (LefterisJP) [inline .github/ISSUE_TEMPLATE]: perhaps mention here if it's built from source show the hash of the commit? like `git show HEAD`?

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
 If it's a bug please describe what was the unexpected thing that occured and what was the
 expected behaviour.
 
-## Solution
-
-Here provide a detailed explanation, if you can suggest one, for how to go about solving the problem.
-
-
-## Tasklist
```
Comment (LefterisJP) [inline .github/ISSUE_TEMPLATE]: As we discussed the tasklist and solution approach should be for our own internal issues and not the bugs, so perhaps move it to the contributing guide?

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (hackaugusto) [reply .github/ISSUE_TEMPLATE]:
```
raiden version
{
  "raiden": "0.2.0+git.r129fa28a",
  "python_implementation": "CPython",
  "python_version": "3.6.4",
  "system": "Linux 64bit_ 4.14.14-1-ARCH x86_64"
}
```

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (hackaugusto) [reply .github/ISSUE_TEMPLATE]: ?

File: .github/ISSUE_TEMPLATE (hunk 3 above)
Comment (hackaugusto) [reply .github/ISSUE_TEMPLATE]: sure

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE]: You have a typo because you like to party ;) . Look at the diff.

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE]: Hmm good point but I meant it also as an alternative in the case they have not managed to make raiden run, so `raiden version` would not work.

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (hackaugusto) [reply .github/ISSUE_TEMPLATE]: AHHh, I totally missed it, thanks.

File: .github/ISSUE_TEMPLATE (hunk 3 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE]: Perhaps the creation of a task list with the tasks that the issue creator thinks needs to be completed before the issue is considered complete.

Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE]: @hackaugusto I will merge this PR now and either you or me can add the deleted part from the issue template to the guide. Reason is: We need it now. Yet again, someone mis-used the template to create their issue and it gets on my nerves.

**Pull request merged**